frontend/src/metabase/plugins/index.ts

```diff
@@ -350,3 +350,7 @@ export const PLUGIN_DASHBOARD_HEADER = {
 export const PLUGIN_QUERY_BUILDER_HEADER = {
   extraButtons: (_question: Question) => [],
 };
+
+export const PLUGIN_BUILD = {
```

Comment:
maybe we can make this name a little more precise? Something like `PLUGIN_IS_EE_BUILD` or something to help devs discover this a little more easily

Reply:
Thanks, yeah I don't think we'll put other things in that plugin so it makes sense